pdns/bpf-filter.cc

```diff
@@ -312,18 +312,17 @@ static FDWrapper loadProgram(const struct bpf_insn* filter, size_t filterSize)
   return fd;
 }
 
-
-BPFFilter::BPFFilter(const BPFFilter::MapConfiguration& v4, const BPFFilter::MapConfiguration& v6, const BPFFilter::MapConfiguration& qnames, BPFFilter::MapFormat format, bool external): d_mapFormat(format), d_external(external)
+BPFFilter::BPFFilter(std::unordered_map<std::string, MapConfiguration> configs, BPFFilter::MapFormat format, bool external)
```

Comment:
you removed initializers from here, was that intentional?

Reply:
Sorry, this was an oversight.